javascript-objects/main.js

```diff
@@ -0,0 +1,36 @@
+const student = {
+  firstName: 'Kyle',
+  lastName: 'Foral',
+  age: 26
+};
+const fullName = student.firstName + ' ' + student.lastName;
+console.log('The value of fullName ', fullName);
+
+student.livesInIrvine = false;
+student.previousOccupation = 'Solar Installer';
+
+console.log('The value of livesInIrvine ', student.livesInIrvine);
+console.log('The value of previousOccupation ', student.previousOccupation);
+
```

Comment:
You should also be console logging the whole `student` object here

Reply:
![image](https://user-images.githubusercontent.com/126129416/226490222-e8dd8881-fa31-44d7-84ff-99641f87422d.png)